mangaki/mangaki/utils/common.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from mangaki.utils.chrono import Chrono
+import pickle
+import os.path
+
+
+PICKLE_DIR = os.path.join(settings.BASE_DIR, '../pickles')
+
+
+class RecommendationAlgorithm(object):
```

Comment:
No need to inherit from `object`. In Python 3, all classes inherits from `object` automatically.